Libs/DeathNotificationLib/DeathNotificationLib.lua

```diff
@@ -116,6 +117,29 @@ local environment_damage = {
 	[-7] = "Slime",
 }
 
+last_duel_to_death_player = nil
```

Comment:
can we prefix these globals with deathlog_?

Reply:
Sure thing.